### DISCUSSION TITLE
Running several emulators with hot reload at once

### DISCUSSION THREAD

**Discussion by Gynko** [unoplatform/uno#15292] [Q&A] [answered]
Hello,
Is it possible to run several emulators at once with hot reload enabled? We are using .NET MAUI.
We would need to see our results on for example 2 different ios phones, and 2 android ones.
Thanks in advance.

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question.

At this time, this is not a scenario that is properly supported by Visual Studio. It _may_ be possible to achieve this using VS Code on a mac with a bit of build rigor in order to get all the apps deployed at the same time without modifications. Yet, [we're still waiting for updates on .NET 8.0.102 or 8.0.200](https://github.com/unoplatform/uno/discussions/15044) that will fix hot reload issues.
